songpal/device.py

```diff
@@ -464,7 +465,7 @@ async def listen_notifications(
         tasks = []
 
         async def handle_notification(notification: ChangeNotification) -> None:
-            callbacks = self.callbacks.get(type(notification), [fallback_callback])
+            callbacks = self.callbacks.get(type(notification), {fallback_callback})
```

Comment:
Is there a specific reason for this change?

Reply:
Oh I left that in to match the type of the callbacks as stored in the class.

Reply:
Ah, okay, that's fine :+1: